Minor updates to Password Import and Registration Hook API references (#4645)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

## Sample SSR responses

> The external service responds to Okta indicating whether to accept the end user's self-registration attempt, and if required update a user profile value. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to self-register and the user profile attribute to update.

Reviewer: ```suggestion
The external service responds to Okta indicating whether to accept the end user's self-registration attempt and update a user profile value, if required. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed to self-register, and also indicates the user profile attribute to update.
```

Author: @susanharper-okta In this case, a user profile attribute is only updated if the self-registration is allowed. So, an external service can respond that a user is allowed or denied. And that's all. Or it can respond that it is allowed (or not because by default it's allowed) and, additionally, if an attribute requires an update.

So recast the blurb as follows:

`The external service responds to Okta indicating whether to accept the end user's self-registration attempt. If self-registration is allowed, an update to a user profile attribute can also be returned. The `commands` object in the body of the HTTPS response contains specific syntax that configures these operations.`
